e2e/tests/prometheus_query.rs

```diff
@@ -0,0 +1,66 @@
+use chrono;
+use failure::{Fallible, ResultExt};
+use prometheus_query::v1::queries::{QueryData, QueryResult, VectorResult};
+use prometheus_query::v1::Client;
+#[test]
+fn query_prometheus() -> Fallible<()> {
+    let api_base = std::env::var("PROM_ENDPOINT").context("PROM_ENDPOINT not set")?;
+    let token = std::env::var("PROM_TOKEN").context("PROM_TOKEN not set")?;
+
+    let client = Client::builder()
+        .api_base(Some(api_base))
+        .access_token(Some(token))
+        .build()?;
+
+    let query = r#"up{job="apiserver"}"#;
+
+    let expected_json = r#"
+        {"status":"success","data":{"resultType":"vector","result":[{"metric":{"version":"4.0.0-0.3"},"value":[1552056334,"0.04081632653061224"]},{"metric":{"version":"4.0.0-0.alpha-2019-03-05-054505"},"value":[1552056334,"0.056451612903225805"]},{"metric":{"version":"4.0.0-0.6"},"value":[1552056334,"0.07692307692307693"]},{"metric":{"version":"4.0.0-0.7"},"value":[1552056334,"0.04519774011299435"]},{"metric":{"version":"4.0.0-0.5"},"value":[1552056334,"0.0625"]}]}}
```

Comment:
I understand this is a fixture which we can't reproduce at runtime because the versions and timestamps will vary.